phase2/ignition/configure-vm.sh

```diff
@@ -0,0 +1,37 @@
+# This is not meant to run on its own, but extends phase1/gce/configure-vm.sh
```

Comment:
can we leave this in phase1/gce/ and rename it to ignition-configure-vm.sh. Code in phase2/ is meant to be platform/cloud independent.